<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Location</class>
 <widget class="QWidget" name="Location">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Location Configuration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #0a0e14;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>SpaceMono Nerd Font</family>
       <pointsize>40</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #4fc3f7;
        padding-bottom: 15px;
        border-bottom: 2px solid #1e1e1e;
        margin-bottom: 20px;</string>
     </property>
     <property name="text">
      <string>LOCATION CONFIGURATION</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>SpaceMono Nerd Font</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #64d2ff;
        padding-left: 10px;</string>
     </property>
     <property name="text">
      <string>SELECT YOUR LOCATION</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="countryList">
     <property name="font">
      <font>
       <family>SpaceMono Nerd Font</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QListWidget {
          background-color: #1e1e1e;
          color: #ffffff;
          border: 2px solid #4fc3f7;
          border-radius: 5px;
          padding: 8px;
        }
        QListWidget::item {
          padding: 10px;
          border-bottom: 1px solid #3a3a3a;
        }
        QListWidget::item:selected {
          background-color: #4fc3f7;
          color: #000000;
        }
        QScrollBar:vertical {
          background: #1e1e1e;
          width: 10px;
        }
        QScrollBar::handle:vertical {
          background: #4fc3f7;
          min-height: 20px;
          border-radius: 5px;
        }
      </string>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>SpaceMono Nerd Font</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #64d2ff;
        padding-left: 10px;
        margin-top: 20px;</string>
     </property>
     <property name="text">
      <string>TIME ZONE:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="timezoneCombo">
     <property name="font">
      <font>
       <family>SpaceMono Nerd Font</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QComboBox {
          background-color: #1e1e1e;
          color: #ffffff;
          border: 2px solid #4fc3f7;
          border-radius: 5px;
          padding: 8px;
          min-height: 30px;
        }
        QComboBox::drop-down {
          border: none;
          width: 30px;
        }
        QComboBox::down-arrow {
          image: url(:/images/arrow-down.png);
          width: 16px;
          height: 16px;
        }
        QComboBox QAbstractItemView {
          background-color: #1e1e1e;
          color: #ffffff;
          selection-background-color: #4fc3f7;
          selection-color: #000000;
          border: 2px solid #4fc3f7;
        }
      </string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="location_btn">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>SpaceMono Nerd Font</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QPushButton {
          color: #64d2ff;
          background-color: rgba(30, 30, 30, 0.7);
          border: 2px solid #4fc3f7;
          border-radius: 5px;
          padding: 10px 40px;
          margin: 20px;
        }
        QPushButton:hover {
          background-color: rgba(79, 195, 247, 0.2);
        }
        QPushButton:pressed {
          background-color: rgba(79, 195, 247, 0.4);
          border-color: #64d2ff;
        }
      </string>
     </property>
     <property name="text">
      <string>NEXT</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
